esmvaltool/config-references.yml

```diff
@@ -523,13 +527,6 @@ authors:
     name: Tilmes, Simone
     institute: NCAR, US
     email: tilmes 'at' ucar.edu
-  lova_to:
-    name: Lovato, Tomas
-    institute: CMCC, IT
-    email: tomas.lovato 'at' cmcc.it
-
-
-
```

Comment:
why did you remove poor old @tomaslovato ?

Reply:
He's a nice guy, but we need only one of him ;-)